tests/integration/test_pfcoil_int.py

```diff
@@ -35,241 +37,241 @@ def pfcoil():
-    monkeypatch.setattr(pfv, "awpoh", 4.232)
-    monkeypatch.setattr(pfv, "oh_steel_frac", 5.926e-1)
-    monkeypatch.setattr(pfv, "bmaxoh0", 1.4e1)
-    monkeypatch.setattr(pfv, "rjohc", 4.070e7)
-    monkeypatch.setattr(pfv, "tmargoh", 1.5)
-    monkeypatch.setattr(pfv, "ipfres", 0)
-    monkeypatch.setattr(pfv, "rjpfalw", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "pfclres", 2.8e-8)
-    monkeypatch.setattr(pfv, "vf", np.full(22, 0.3))
-    monkeypatch.setattr(pfv, "ric", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "bpf", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "jscoh_eof", 4.758e8)
-    monkeypatch.setattr(pfv, "zpf", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "rb", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "ra", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "jscoh_bop", 3.562e8)
-    monkeypatch.setattr(pfv, "cptdin", np.full(22, 4.22e4))
-    monkeypatch.setattr(pfv, "pfcaseth", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "rpf", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "cohbop", 1.693e7)
-    monkeypatch.setattr(pfv, "zh", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "wtc", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "zl", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "turns", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "wts", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "a_oh_turn", 0.0)
-    monkeypatch.setattr(tfv, "dcond", np.full(9, 9.0e3))
-    monkeypatch.setattr(tfv, "tftmp", 4.750)
-    monkeypatch.setattr(tfv, "tcritsc", 1.6e1)
-    monkeypatch.setattr(tfv, "str_cs_con_res", -5.000e-3)
-    monkeypatch.setattr(tfv, "fhts", 0.5)
-    monkeypatch.setattr(tfv, "bcritsc", 2.4e1)
-    monkeypatch.setattr(tfv, "b_crit_upper_nbti", 1.486e1)
-    monkeypatch.setattr(tfv, "t_crit_nbti", 9.04)
-    monkeypatch.setattr(constants, "dcopper", 8.9e3)
-
-    # Mocks for peakb()
-    monkeypatch.setattr(bv, "iohcl", 1)
-    monkeypatch.setattr(pfv, "waves", np.full([22, 6], 0.0))
-    monkeypatch.setattr(pfv, "ngrp", 4)
-    monkeypatch.setattr(pfv, "ncls", np.array([1, 1, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0]))
-    monkeypatch.setattr(pfv, "curpfb", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "curpff", np.full(22, 0.0))
-    monkeypatch.setattr(pfv, "curpfs", np.full(22, -175.84911993600002))
-    monkeypatch.setattr(pv, "rmajor", 8.938)
-    monkeypatch.setattr(pv, "plascur", 1.8254e7)
-
-    # Mocks for hoop_stress()
-    monkeypatch.setattr(tfv, "poisson_steel", 3.0e-1)
-
-    # Mocks for superconpf()
-    monkeypatch.setattr(eh, "fdiags", np.full(8, -9.99999e5))
-    monkeypatch.setattr(tfv, "tmargmin_cs", 1.5)
-    monkeypatch.setattr(tfv, "temp_margin", 0.0)
-    monkeypatch.setattr(tfv, "b_crit_upper_nbti", 1.486e1)
-    monkeypatch.setattr(tfv, "b_crit_upper_nbti", 9.04)
-
-    pfcoil.ohcalc()
-
-    assert pytest.approx(pfv.bpf[4]) == 9.299805e2
-    assert pytest.approx(pfv.rjohc) == -7.728453e9
-
-
-def test_efc(pfcoil, monkeypatch):
+# def test_pfcoil(monkeypatch, pfcoil):
```

Comment:
As all this is commented back in later, can this be fixed up to reduce churn in this PR?